Latest ewebkit snapshot depends on yet-unstable glib/libsoup

--- origsrc/elementary-1.7.0/src/lib/elm_web.c	2012-08-20 18:01:42.000000000 -0500
+++ src/elementary-1.7.0/src/lib/elm_web.c	2012-09-12 04:26:05.613324500 -0500
@@ -1448,12 +1448,12 @@ elm_web_title_get(const Evas_Object *obj
    ELM_WEB_CHECK(obj) NULL;
 
 #ifdef HAVE_ELEMENTARY_WEB
-   const Ewk_Text_With_Direction *txt;
+   const char *txt;
 
    ELM_WEB_DATA_GET(obj, sd);
 
    txt = ewk_view_title_get(ELM_WIDGET_DATA(sd)->resize_obj);
-   if (txt) return txt->string;
+   if (txt) return txt;
 
    return NULL;
 #else
